docs/src/main/sphinx/release/release-405.md

```diff
@@ -0,0 +1,90 @@
+# Release 405 (23 Nov 2022)
+
+```{warning}
+Where's release 404? Not found.
+```
+
+## General
+
+* Improve performance of queries that compare `date` columns with
+  `timestamp with time zone` literals. ({issue}`5798`)
+* Improve performance and resource utilization of `INSERT` queries into
+  partitioned and bucketed tables. ({issue}`14718`)
+
+## Security
+
+* Add support for string replacement as part of
+  [impersonation rules](system-file-auth-impersonation-rules). ({issue}`14962`)
+* Fix some `system.metadata` tables improperly showing the names of catalogs
+  which the user cannot access. ({issue}`14000`)
+* Fix `USE` statement improperly showing the names of catalogs and schemas which
```

Comment:
`USE` didn't "show" them - it disclosed that they exist because the error message said "access denied" instead of something more vague.